tests/test_run_onboarding.py

```
@@ -359,6 +359,73 @@ def test_setup_and_secret_renewal_commands_are_available():
+    _write_executable(
+        fake_bin / "tailscale",
+        "#!/usr/bin/env bash\n"
+        'case "$1" in\n'
+        '  status) echo \'{"BackendState":"Running"}\' ;;\n'
+        '  debug) echo "{\"OperatorUser\":\"$(id -un)\"}" ;;\n'
+        '  funnel) if [ "$2" = "status" ]; then echo \'{"Funnel":{"On":true}}\'; fi ;;\n'
+        "esac\n"
+        "exit 0\n",
+    )
+
+    result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)